<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>offline</class>
 <widget class="QWidget" name="offline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="eng1">
      <property name="text">
       <string>CET4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng2">
      <property name="text">
       <string>CET6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng3">
      <property name="text">
       <string>GMAT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng4">
      <property name="text">
       <string>GRE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng5">
      <property name="text">
       <string>IELTS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng6">
      <property name="text">
       <string>SAT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng7">
      <property name="text">
       <string>TOFEL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng8">
      <property name="text">
       <string>考研</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng9">
      <property name="text">
       <string>专八</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="eng10">
      <property name="text">
       <string>专四</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="jap1">
      <property name="text">
       <string>日本語</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="word">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>word</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="chinesemeaning">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>中释</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="languagemeaning">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>language meaning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
